plugins/kubernetes/app/views/kubernetes/roles/_form.html.erb

```diff
@@ -18,8 +18,10 @@
       </div>
     <% end %>
 
-    <%= form.input :autoscaled, label: "Replicas managed externally", as: :check_box, help: "Deploying will not change replicas count. Use this when autoscaling deployments." %>
-    <%= form.input :blue_green, as: :check_box, label: "Blue/Green Deployment", help: "Starts a new isolated deployment shifting between blue and green sufixes, switching service selectors if successfully deployed and deleting previous resources, all active resources must be deleted manually when switching this" %>
+    <%= form.input :autoscaled, label: "Replicas managed externally", as: :check_box,
+          help: "<ul><li>Deploying will not change replicas count</li><li>Replica counts are used as the minimum number of replicas that need to be live</li><li>Use when autoscaling Deployments</li></ul>".html_safe %>
```

Comment:
> […] change replicas count\</li>\<li>Replica counts are

Which is the correct plural form? “Replica counts” or “replicas count”?

Reply:
`replica counts`